site/main.css

```diff
@@ -12,7 +12,12 @@ h1, h2 {
 }
 
 a {
-	color: #00897b;
+	color: #6F6F6FFF;
```

Comment:
8 digits color? Is that intentional? I don't see a reason to use rgba instead of rgb if the alpha is practically not used. Am I missing anything?

Reply:
Was copy  paste from color picker, no reason to have rgba, well spotted :)